nn/naming.py

```diff
@@ -705,12 +710,19 @@ def get_ext_net_dict_py_code_str(self, root_module: nn.Module, *, with_imports:
     net_dict = dim_tags_proxy.collect_dim_tags_and_transform_config(net_dict)
     imports = {}
     net_dict = self._post_process_transform(net_dict, imports=imports)
-
     code_lines = []
+
     if with_imports:
       code_lines.append(self._ImportPyCodeStr + "\n")
     for import_str in imports:
       code_lines.append(import_str + "\n")
+
+    if ref_extern_data_dims_via_global_config:
+      code_lines += ["from returnn.config import get_global_config\n",
+                     "config = get_global_config()\n"]
```

Comment:
Do not use hanging indents (same rule as in RETURNN).
```suggestion
      code_lines += [
        "from returnn.config import get_global_config\n",
        "config = get_global_config()\n"]
```